stable/spring-cloud-data-flow/README.md

```diff
@@ -30,11 +30,30 @@ If you are using a cluster that does not have a load balancer (like Minikube) th
 $ helm install --name my-release --set server.service.type=NodePort stable/spring-cloud-data-flow
 ```
 
-RabbitMQ is the default messaging layer, Kafka can be enabled instead by using the following `set` flags to the `helm` command, for example:
+### Data Store
+
+By default, MySQL is deployed with this chart. However, if you wish to use an external database, please use the following `set` flags to the `helm` command to disable MySQL deployment, for example:
+
+`--set mysql.enabled=false`
+
+In addition, you are required to set all fields listed in [External Database Configuration](#external-database-configuration-mysqlenabledfalse).
+
+### Messaging Layer
+
+There are three messaging layer available in this chart:
+- RabbitMQ (default)
+- RabbitMQ HA
+- Kafka
+
+To change the messaging layer to a highly available (HA) version of RabbitMQ, use the following `set` flags to the `helm` command, for example:
+
+`--set rabbitmq-ha.enabled=true,rabbitmq.enabled=false`
+
+Alternatively, to change the messaging layer to Kafka, use the following `set` flags to the `helm` command, for example:
 
 `--set kafka.enabled=true,rabbitmq.enabled=false`
 
-Only one messaging layer can be used at a given time. If RabbitMQ and Kafka are enabled, both charts will be installed with RabbitMQ being used in the deployment.
+Only one messaging layer can be used at a given time. If two or more messaging layers are enabled, the enabled charts will be installed but only one will be used. The order of preference is given by (preferred) RabbitMQ HA > RabbitMQ > Kafka (least preferred).
```

Comment:
please remove `The order of preference is given by (preferred) RabbitMQ HA > RabbitMQ > Kafka (least preferred).`. realistically, the selection of middleware is determined by `set` flags and if not provided, defaults from values file which is documented in the README.md

Reply:
The order of preference is when two or more messaging layers are enabled. I have made it more explicit in the readme, please let me know if it's ok